measures/measures-schema.yaml

```diff
@@ -110,6 +110,14 @@ definitions:
         description: Possible methods for submitting performance data for the measure.
         type: array
         items: { $ref: #/definitions/methods }
+      eligibilityOptions:
+        description: Eligibility options mirror denominator options in measure specifications. Each option comprises a set of codes used to identify eligible instances of the associated measure.
```

Comment:
Sure! There are different types of specifications (for example, eCQM specifications versus QCDR specifications). It would be helpful to be explicit when you say "mirror denominator options in measure specifications" that you mean "QCDR measure specifications" because right now these (a) only apply to claims (b) may confuse someone if they are looking at an eCQM spec and can't find mention of "denominator options".